src/menu.tsx

```diff
@@ -91,38 +92,49 @@ export function get_menu_items(
             }
         );
     } else if (selected.length === 1) {
+        const item = selected[0];
+
+        if (item.type === 'reg' && item.size && item.size < 1000000) // 1MB
+            menuItems.push(
+                {
+                    id: "open-file",
+                    title: _("Open text file"),
```

Comment:
How do you know it's a text file? There's no check if it's a binary file. That would be a good idea, but other than that it feels less confusing to just say "Edit file"?